src/eth/executor.rs

```diff
@@ -14,12 +21,17 @@ use crate::eth::primitives::TransactionExecution;
 use crate::eth::primitives::TransactionInput;
 use crate::eth::storage::EthStorage;
 
-/// High-level coordinator of Ethereum transactions.
+/// The EthExecutor struct is responsible for orchestrating the execution of Ethereum transactions.
```

Comment:
This EthExecutor internal working will change soon. It will use multi-producer multi-consumer (mpmc) channels with a pool of EVMs, so we can:

* avoid locks
* allow multiple transactions to execute in parallel

But for now it is ok because it is describing what we have here.

Reply:
mpmc will be awesome